scripts/automated-tools/start.sh

```
@@ -0,0 +1,12 @@
+#!/bin/bash
+export PATH=/opt/alt/alt-nodejs20/root/usr/bin:$PATH
+cd /home/u875310796/domains/cortexbuildpro.com/public_html/__backend__
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)